packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -652,6 +714,7 @@ aar artifact already exists, continuing...''',
 
     final zippedRunner = await Directory(runnerPath).zipToTempFile();
     try {
+      logger.detail('[archive] zipped runner.app to ${zippedRunner.path}');
```

Comment:
Do we want this log? We don't have an equivalent for other platforms so just wondering if this was left in intentionally.

Reply:
I don't feel strongly. Users will only see it when running with the `-v` flag, and there's so much other output that I'm not sure it matters much one way or the other.